src/components/CardComponentList.tsx

```diff
@@ -1,46 +1,45 @@
-        <div className="xl:h-[50vh] lg:h-[60vh] overflow-hidden">
-            {/* remember to have a default height/width if using media breakpoints */}
-            <div className="overflow-y-scroll lg:h-[50vh] md:h-[60vh] h-[80vh]">
-                <ul className="divide-y divide-gray-200">
-                    {data.items.map((item, index) => (
-                        <li key={index} className="py-4">
-                            <div className="flex flex-row items-center">
+  const isMobile = useMediaQuery({ query: "(max-width: 767px)" });
+  return (
+    <div className="xl:h-[50vh] lg:h-[60vh] overflow-hidden">
+      {/* remember to have a default height/width if using media breakpoints */}
+      <div className="overflow-y-scroll lg:h-[50vh] md:h-[60vh] h-[80vh]">
+        <ul className="divide-y divide-gray-200">
+          {data.items.map((item, index) => (
+            <li key={index} className="py-4">
+              <div className="flex flex-row items-center">
+                <span className="mr-2">{isMobile ? index + 1 + "." : index + 4 + "."}</span>
 
-                                <span className="mr-2">
-                                    {isMobile ? (index + 1) + "." : (index + 4) + "."}
-                                </span>
+                <Image
+                  src={item.album.images[0].url}
+                  alt={item.name}
+                  width={100}
+                  height={100}
+                  className="aspect-square"
+                />
 
-                                <Image src={item.album.images[0].url} alt={item.name} width={100} height={100} className="aspect-square" />
-
-                                <div className="flex-grow">
-                                    <div className="text-center">
-                                        {item.name}
-                                    </div>
-                                    <div className="flex flex-wrap justify-center py-1">
-                                        {item.artists.map((artist, index) => (
-                                            <div key={index} className="mx-2 text-orange-400">
-                                                {artist.name}
-                                            </div>
-                                        ))}
-                                    </div>
-                                    <div className="text-center">
-                                        {item.album.name}
-                                    </div>
-                                </div>
-                            </div>
-                        </li>
+                <div className="flex-grow lg:mr-32">
+                  <div className="text-center text-xs md:text-lg">{item.name}</div>
+                  <div className="flex flex-wrap justify-center py-1">
+                    {item.artists.map((artist, index) => (
+                      <div key={index} className="mx-2 text-orange-400 text-xs md:text-lg">
```

Comment:
Index as a rule of thumb is not a good choice for a key.

https://react.dev/learn/rendering-lists#rules-of-keys

Reply:
changed to using its unique id instead